### PR TITLE
oop

### PR DESCRIPTION
**Pull request by IlyaGorod**

### CODE REVIEW

File: hotelBooking/src/SingleRoom.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+public class SingleRoom extends Hotel implements PricePerRoom{
```
Comment (nmelnyk-corevalue) [inline hotelBooking/src/SingleRoom.java:1]: SingleRoom extends Room implements Priceable - it's better

File: hotelBooking/src/Adult.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+public class Adult extends Person implements Human{
+
+    protected Adult(int age, int pay) {
+        super(age,pay);
+    }
+
+    public void getHuman() {
+        System.out.println(this.pay);
+    }
```
Comment (nmelnyk-corevalue) [inline hotelBooking/src/Adult.java:9]: getHuman() - but print Pay... a little misunderstanding, what do u think?

Comment (IlyaGorod) [reply hotelBooking/src/Adult.java:9]: I agree

File: hotelBooking/src/SingleRoom.java (hunk 1 above)
Comment (IlyaGorod) [reply hotelBooking/src/SingleRoom.java:1]: ok! th u

Comment (IlyaGorod) [reply hotelBooking/src/SingleRoom.java:1]: gorod_oop - this is my branch, did it yesterday

Comment (nmelnyk-corevalue) [reply hotelBooking/src/SingleRoom.java:1]: https://prnt.sc/pwxap2

Comment (nmelnyk-corevalue) [reply hotelBooking/src/SingleRoom.java:1]: make PR with your new branch and delete unnecessary PRs - pliz have a look on screenshot above.

**Pull request closed without merging**